plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/pluggabletask/PluggableTaskConfigStore.java

```diff
@@ -18,8 +18,8 @@
 
 import com.thoughtworks.go.plugin.access.config.PluginPreferenceStore;
 
-public final class PluggableTaskConfigStore extends PluginPreferenceStore<TaskPreference> {
```

Comment:
My guess is that the class is being mocked in the TaskViewServiceTest. Can't mock when it is final. Classes shouldn't be final, IMHO.